application/server.py

```diff
@@ -66,8 +72,23 @@ def handle_create_room_form():
         confirmed_password = request.form.get('confirmed-room-password')
 
         if room_password == confirmed_password:
-            # TODO: Add game/room to database
-            # TODO: Add user to database (this will require using the game's id)
+            game_object = models.Game(room_name=room_name, room_password=room_password)
+            db.session.add(game_object)
+            db.session.commit()
+
+            # color = randrange(1,255)
+            newp = models.Player(name=name, color="#CCFF00", game_id=game_object.id)
+            db.session.add(newp)
+            db.session.commit()
+
+            players = models.Player.query.all()
+            for player in players:
```

Comment:
Once you've moved the _repr_ functions to the right place you'll be able to just calls print(players)